src/WorkflowCore/Interface/IWorkflowDefinitionValidator.cs

```diff
@@ -0,0 +1,9 @@
+using WorkflowCore.Models;
+
+namespace WorkflowCore.Interface
+{
+    public interface IWorkflowDefinitionValidator
+    {
+        bool IsDefinitionValid(WorkflowDefinition definition);
```

Comment:
is this part of the try/catch feature? or something else?

Reply:
Right now this only checks that every try has corresponding catch.